Firestore/Example/Firestore.xcodeproj/project.pbxproj

```diff
@@ -717,6 +719,7 @@
 		54740A561FC913EB00713A1A /* util */ = {
 			isa = PBXGroup;
 			children = (
+				B68B1E002213A764008977EF /* to_string_apple_test.mm */,
```

Comment:
If you run `scripts/sync_project.rb` does this change?

Reply:
Done.